.travis.yml

```
@@ -427,7 +427,7 @@ jobs:
     <<: *reset-prerequisites
     stage: Deploy
     name: Publishing current Git tagged version of dist to PyPI
-    if: repo == "ansible/molecule" AND tag IS present
+    if: repo == "ansible/molecule" AND tag IS present AND type == push
```

No, this is not needed. Those deploy jobs appear there because of push events in branches since you did pushes to branches in upstream. Tag builds are only triggered by pushes so it doesn't matter here.

```suggestion
    if: repo == "ansible/molecule" AND tag IS present
```